examples/jsm/loaders/3DMLoader.js

```diff
@@ -842,7 +856,7 @@ function Rhino3dmWorker() {
 
 			const _object = objs.get( i );
 
-			const object = extractObjectData( _object, doc );
+			const object = extractObjectData( _object, doc, ( warning )=>warnings.push( warning ) );
```

Comment:
I believe this could be avoided by collecting warnings outside of the worker by calling `self.postMessage()` from inside the worker and adding a warning case when setting up the worker..